pennylane/labs/tests/vibrational/test_pes_generator.py

```diff
@@ -56,8 +55,8 @@ def test_import_mpi4py(monkeypatch):
                 "norm_mode": np.array(
                     [
                         [
-                            [-1.93180788e-17, -2.79300482e-17, -9.70607797e-01],
-                            [-3.49162986e-17, 8.71500605e-17, 5.14976259e-02],
+                            [-1.93180788e-17, -2.79300482e-17, 9.70607797e-01],
+                            [-3.49162986e-17, 8.71500605e-17, -5.14976259e-02],
```

Comment:
Are these (and below) numerical values changed due to the modified constant values, or something else?

Reply:
Yes, these are changing because of different precision being used in scipy constants.